src/components/article/CommentCard.jsx

```diff
@@ -1,15 +1,68 @@
 import "./Article.css";
-import  dayjs  from 'dayjs'
-import advancedFormat from 'dayjs/plugin/advancedFormat'
+import dayjs from "dayjs";
+import advancedFormat from "dayjs/plugin/advancedFormat";
+import { useContext, useState } from "react";
+import { MdDelete } from "react-icons/md";
+import { UsernameContext } from "../../contexts/Username";
+import { deleteComment } from "../api";
+
+export default function CommentCard({ comment, handleViewComments }) {
+  dayjs.extend(advancedFormat);
+  const { username } = useContext(UsernameContext);
+  const [isDeleting, setIsDeleting] = useState(false);
+  const [isDeleted, setIsDeleted] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  function handleDeleteComment() {
+    setIsDeleting(true);
+    deleteComment(comment.comment_id)
+      .then(() => {
+        setIsDeleting(false);
+        setIsDeleted(true);
+        handleViewComments();
+      })
+      .catch(() => {
+        setIsError(true);
+      });
+  }
+
+  function deleteBtn() {
```

Comment:
Not ideal to have components declared inside of other components rather than give them their own file and import them in. It bloats your component and limits your ability to reuse components across your project